### PR TITLE
Formatted types/e* packages with dprint

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
> 👉 Partial implementation of https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993. If you're seeing this and are surprised it's happening, please read the discussion and give feedback! ❤️ 

Implements the dprint formatting changes packages whose names start with `e`. We're splitting up the changes to make it a bit easier to apply & review them en masse - and, if we need later on, revert / bisect for issues. These changes generated with roughly:

```shell
git checkout dprint-onboarding -- .dprint.jsonc
time npx dprint fmt "types/e*/**/*.t*"
# Formatted 1665 files.
# npx dprint fmt "types/e*/**/*.t*"  7.69s user 1.02s system 328% cpu 2.647 total
rm .dprint.jsonc
```

~CI failures are a combination of existing errors and due to [DefinitelyTyped-tools > feat: remove formatting rules rules from dtslint](https://github.com/microsoft/DefinitelyTyped-tools/pull/699) not being ~merged~ included in CI yet.~ ✔️ 

CI failures are from existing errors.

Note that I manually added a couple of newlines in `types/ember/test/create.ts` and `types/ember/v3/test/create.ts` to avoid:

```plaintext
Error formatting /Users/josh/repos/DefinitelyTyped/types/ember/test/create.ts. Message: Expected ';', '}' or <eof> at /Users/josh/repos/DefinitelyTyped/types/ember/test/create.ts:14:15

  assertType<<K extends keyof Ember.Object>(key: K) => UnwrapComputedPropertyGette...
                ~~~~~~~
Error formatting /Users/josh/repos/DefinitelyTyped/types/ember/v3/test/create.ts. Message: Expected ';', '}' or <eof> at /Users/josh/repos/DefinitelyTyped/types/ember/v3/test/create.ts:14:15

  assertType<<K extends keyof Ember.Object>(key: K) => UnwrapComputedPropertyGette...
                ~~~~~~~
```

Includes:
* #66357

### CODE REVIEW

File: types/easeljs/easeljs-tests.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
-
-
 var stage: any;
 var myContext2D: any;
 
 function test_simple() {
-    var canvas = <HTMLCanvasElement>document.getElementById('canvas');
+    var canvas = <HTMLCanvasElement> document.getElementById("canvas");
```
Comment (robertIsaac) [inline types/easeljs/easeljs-tests.ts:5]: why did you reformat all the files? I think these changes are not desired
also it makes it very difficult to review your PR

**Review by robertIsaac — CHANGES_REQUESTED** [review]

**Comment by robertIsaac** [comment]
can you configure dprint to match the current style, 
what I see most notable is replacing all single quote into double quotes

**Review by HichemTab-tech — APPROVED** [review]
Small changes, the codes still looks fine.

**Review by sheetalkamat — APPROVED** [review]

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## each ([<kbd>unpkg</kbd>](https://unpkg.com/browse/each@latest/))
was missing the following properties:
1. Each
## easy-session ([<kbd>unpkg</kbd>](https://unpkg.com/browse/easy-session@latest/))
was missing the following properties:
1. can
1. can
## easy-xapi-utils ([<kbd>unpkg</kbd>](https://unpkg.com/browse/easy-xapi-utils@latest/))
was missing the following properties:
1. setRbac
1. rbac
## elasticsearch ([<kbd>unpkg</kbd>](https://unpkg.com/browse/elasticsearch@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'elasticsearch'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. ConnectionPool
1. Transport
## electron-spellchecker ([<kbd>unpkg</kbd>](https://unpkg.com/browse/electron-spellchecker@latest/))
was missing the following properties:
1. DictionarySync
1. SpellChecker
1. setGlobalLogger
## elementtree ([<kbd>unpkg</kbd>](https://unpkg.com/browse/elementtree@latest/))
was missing the following properties:
1. PI
1. ElementPath
## elgamal ([<kbd>unpkg</kbd>](https://unpkg.com/browse/elgamal@latest/))
was missing the following properties:
1. Errors
## ember-changeset-validations ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-changeset-validations@latest/))
was missing the following properties:
1. name
## ember-cli-fastboot ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-cli-fastboot@latest/))
was missing the following properties:
1. name
1. init
1. existsSync
1. included
1. importTransforms

<details>
<summary>as well as these 5 other properties...</summary>
<p>contentFor, treeForFastBoot, treeForPublic, serverMiddleware, postBuild</p>
</details>

## ember-feature-flags ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-feature-flags@latest/))
was missing the following properties:
1. name
1. name
## ember-mocha ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-mocha@latest/))
was missing the following properties:
1. name
1. init
1. postBuild
1. checkPackages
1. included

<details>
<summary>as well as these 6 other properties...</summary>
<p>targetOptions, contentFor, treeForVendor, treeForAddonTestSupport, overrideTestCommandFilter, setTestGenerator</p>
</details>

## ember-modal-service ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-modal-service@latest/))
was missing the following properties:
1. name
1. name
## ember-task-scheduler ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-task-scheduler@latest/))
was missing the following properties:
1. name
1. importPolyfill
1. included
1. name
1. importPolyfill

<details>
<summary>as well as these 1 other properties...</summary>
<p>included</p>
</details>

## ember-useragent ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ember-useragent@latest/))
was missing the following properties:
1. name
1. options
1. name
1. options
## empty-dir ([<kbd>unpkg</kbd>](https://unpkg.com/browse/empty-dir@latest/))
was missing the following properties:
1. isEmpty
## enamdict ([<kbd>unpkg</kbd>](https://unpkg.com/browse/enamdict@latest/))
was missing the following properties:
1. enamdictFile
1. process
1. index
## encoding-down ([<kbd>unpkg</kbd>](https://unpkg.com/browse/encoding-down@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'encoding-down'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
## engine.io-client ([<kbd>unpkg</kbd>](https://unpkg.com/browse/engine.io-client@latest/))
was missing the following properties:
1. priorWebsocketSuccess
1. protocol
1. Transport
1. transports
1. parser

<details>
<summary>as well as these 5 other properties...</summary>
<p>priorWebsocketSuccess, protocol, Transport, transports, parser</p>
</details>

## engine.io ([<kbd>unpkg</kbd>](https://unpkg.com/browse/engine.io@latest/))
was missing the following properties:
1. Transport
1. transports
1. parser
## enigma.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/enigma.js@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'enigma.js'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
## enquire.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/enquire.js@latest/))
was missing the following properties:
1. queries
1. browserIsIncapable
1. constructor
## enzyme-async-helpers ([<kbd>unpkg</kbd>](https://unpkg.com/browse/enzyme-async-helpers@latest/))
was missing the following properties:
1. wait
1. wait
## enzyme-to-json ([<kbd>unpkg</kbd>](https://unpkg.com/browse/enzyme-to-json@latest/))
was missing the following properties:
1. renderToJson
1. shallowToJson
1. mountToJson
1. renderToJson
1. shallowToJson

<details>
<summary>as well as these 1 other properties...</summary>
<p>mountToJson</p>
</details>

## epilogue ([<kbd>unpkg</kbd>](https://unpkg.com/browse/epilogue@latest/))
was missing the following properties:
1. Controllers
1. Controllers
## escodegen ([<kbd>unpkg</kbd>](https://unpkg.com/browse/escodegen@latest/))
was missing the following properties:
1. version
1. browser
1. FORMAT_MINIFY
1. FORMAT_DEFAULTS
## escpos ([<kbd>unpkg</kbd>](https://unpkg.com/browse/escpos@latest/))
was missing the following properties:
1. Bluetooth
1. Adapter
## esri-leaflet ([<kbd>unpkg</kbd>](https://unpkg.com/browse/esri-leaflet@latest/))
was missing the following properties:
1. FeatureManager
1. Identify
1. Support
1. Util
1. VERSION

<details>
<summary>as well as these 6 other properties...</summary>
<p>get, identify, options, post, request, service</p>
</details>

## eth-lightwallet ([<kbd>unpkg</kbd>](https://unpkg.com/browse/eth-lightwallet@latest/))
was missing the following properties:
1. txutils
1. encryption
1. upgrade
## eth-url-parser ([<kbd>unpkg</kbd>](https://unpkg.com/browse/eth-url-parser@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'eth-url-parser'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. apply
## ethereum-blockies ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ethereum-blockies@latest/))
was missing the following properties:
1. render
## ethereumjs-abi ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ethereumjs-abi@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'ethereumjs-abi'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. solidityHexValue
## evaporate ([<kbd>unpkg</kbd>](https://unpkg.com/browse/evaporate@latest/))
was missing the following properties:
1. getLocalTimeOffset
## event-stream ([<kbd>unpkg</kbd>](https://unpkg.com/browse/event-stream@latest/))
was missing the following properties:
1. Stream
1. through
1. from
1. connect
1. pipe

<details>
<summary>as well as these 4 other properties...</summary>
<p>collect, flatmapSync, log, pipeable</p>
</details>

## events ([<kbd>unpkg</kbd>](https://unpkg.com/browse/events@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'events'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. defaultMaxListeners
1. init
1. listenerCount
## exaroton ([<kbd>unpkg</kbd>](https://unpkg.com/browse/exaroton@latest/))
was missing the following properties:
1. ServerStatus
## exif ([<kbd>unpkg</kbd>](https://unpkg.com/browse/exif@latest/))
was missing the following properties:
1. TAGS
## expect.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/expect.js@latest/))
was missing the following properties:
1. Assertion
1. version
1. stringify
1. eql
## expired-storage ([<kbd>unpkg</kbd>](https://unpkg.com/browse/expired-storage@latest/))
was missing the following properties:
1. version
1. author
1. gitUrl
## express-brute-memcached ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-brute-memcached@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'express-brute-memcached'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. defaults
1. The declaration doesn't match the JavaScript module 'express-brute-memcached'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. defaults
## express-brute ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-brute@latest/))
was missing the following properties:
1. defaults
1. instanceCount
1. defaults
1. instanceCount
## express-domain-middleware ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-domain-middleware@latest/))
was missing the following properties:
1. id
1. id
## express-flash ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-flash@latest/))
was missing the following properties:
1. version
## express-formidable ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-formidable@latest/))
was missing the following properties:
1. parse
## express-partials ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-partials@latest/))
was missing the following properties:
1. register
1. renderer
1. lookup
## express-useragent ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-useragent@latest/))
was missing the following properties:
1. getWechatVersion
1. getElectronVersion
1. testWechat
## express-version-request ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-version-request@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'express-version-request'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. setVersionByAcceptFormat
1. isObject
1. removeWhitespaces
## express-version-route ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-version-route@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'express-version-route'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. checkVersionMatch
1. getDefaultRoute
## express-ws-routes ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express-ws-routes@latest/))
was missing the following properties:
1. apply
1. call
1. bind
1. toString
1. length

<details>
<summary>as well as these 11 other properties...</summary>
<p>arguments, caller, name, apply, call, bind, toString, length, arguments, caller, name</p>
</details>

## express ([<kbd>unpkg</kbd>](https://unpkg.com/browse/express@latest/))
was missing the following properties:
1. Route
## eyes ([<kbd>unpkg</kbd>](https://unpkg.com/browse/eyes@latest/))
was missing the following properties:
1. defaults
1. print
1. stylize
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a759bfd35d9b54f32d6e4f2583209fd957d55903
</p>

**Review by johnnyreilly — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
Ready to merge

**Pull request merged**